var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -33,8 +33,7 @@ class Nwchem(Package):
     homepage = "http://www.nwchem-sw.org"
     url      = "http://www.nwchem-sw.org/images/Nwchem-6.6.revision27746-src.2015-10-20.tar.gz"
 
-    version('6.6', 'c581001c004ea5e5dfacb783385825e3',
-            url='http://www.nwchem-sw.org/images/Nwchem-6.6.revision27746-src.2015-10-20.tar.gz')
+    version('6.6.revision27746-src.2015-10-20', 'c581001c004ea5e5dfacb783385825e3')
```

Comment:
I'm not sure this is a good idea.  What will our version parser do to the version '6.6.revision27746-src.2015-10-20'?  Still, I don't think that '6.6' is the right version either.  How about '6.6r27746'?